eventuate-tram-testing-support/src/test/java/io/eventuate/tram/testing/MessageTrackerTest.java

```diff
@@ -39,6 +42,11 @@
     public MessageTracker messageTracker(MessageConsumer messageConsumer) {
       return new MessageTracker(channels, messageConsumer);
     }
+
+    @Bean
+    public EventuateTransactionTemplate eventuateTransactionTemplate(TransactionTemplate transactionTemplate) {
```

Comment:
This is slightly suspicious: why is this @Bean defined in multiple tests rather than in an eventuate-common @Configuration class and  @Imported?

Reply:
I will fix